plugins/modules/aci_maintenance_policy.py

```diff
@@ -36,17 +37,69 @@
     description:
     - The name of scheduler that is applied to the policy.
     type: str
-  adminst:
+  admin_state:
     description:
-    - Will trigger an immediate upgrade for nodes if adminst is set to triggered.
+    - The administrative state of the executable policies.
+    - Will trigger an immediate upgrade for nodes if C(admin_state) is set to triggered.
+    - The APIC defaults to C(untriggered) when unset during creation.
     type: str
     choices: [ triggered, untriggered ]
-    default: untriggered
-  ignoreCompat:
+  download_state:
+    description:
+    - The download state of the executable policies.
+    - The APIC defaults to C(untriggered) when unset during creation.
+    type: str
+    choices: [ triggered, untriggered ]
+  notif_condition:
+    description:
+    - Specifies under what pause condition will admin be notified via email/text as configured.
+    - This notification mechanism is independent of events/faults.
+    - The APIC defaults to C(notifyOnlyOnFailures) when unset during creation.
+    type: str
+    choices: [ notifyAlwaysBetweenSets, notifyNever, notifyOnlyOnFailures ]
```

Comment:
Change choices to snake case if it's not a hassle:

choices: [ notify_always_between_sets, notify_never, notify_only_on_failures ]